crypto/engine/eng_cryptodev.c

```diff
@@ -1247,13 +1247,21 @@ cryptodev_asym(struct crypt_kop *kop, int rlen, BIGNUM *r, int slen,
         return (ret);
 
     if (r) {
-        kop->crk_param[kop->crk_iparams].crp_p = calloc(rlen, sizeof(char));
+        kop->crk_param[kop->crk_iparams].crp_p = OPENSSL_zalloc(rlen);
```

Comment:
If you are changing to `OPENSSL_zalloc` then we also need to change the `free` in `zapparams` to be `OPENSSL_free`